src/mumble/JackAudio.cpp

```diff
@@ -525,6 +529,22 @@ bool JackAudioSystem::disconnectPort(jack_port_t *port) {
 	return true;
 }
 
+bool JackAudioSystem::renamePort(jack_port_t *port, const char *name) {
+	QMutexLocker lock(&qmWait);
+
+	if (!client || !port) {
+		return false;
+	}
+
+	const auto ret = jack_port_rename(client, port, name);
```

Comment:
This shouldn't use `auto` because the type of `ret` is not deducible from the immediate context (and is not an iterator)